README.md

```diff
@@ -12,12 +12,12 @@ An Apollo Link to easily try out GraphQL without a full server. It can be used t
 ## Installation
 
 ```bash
-npm install apollo-link-rest apollo-link graphql graphql-anywhere qs --save
+npm install apollo-link-rest @apollo/client graphql qs --save
 or
-yarn add apollo-link-rest apollo-link graphql graphql-anywhere qs
+yarn add apollo-link-rest @apollo/client graphql qs
```

Comment:
@fbartho I wanted to draw your attention to the diff in https://github.com/apollographql/apollo-link-rest/pull/301/commits/2c0b7dc7e5eb68a2852933d6b95ce1bf7fd1d3f3: I removed `apollo-link` here in favor of `@apollo/client` to reflect the peer dependencies in `package.json`